code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1728,6 +1728,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(FULLNESS_LEVEL_NOMOREPLZ to INFINITY)
 			H.throw_alert("fullness", /obj/screen/alert/beegbelly)
 
+	// Update here for changing belly to match stuffed-ness
+	var/obj/item/organ/genital/belly/B= H.getorganslot("belly")
+	if(!isnull(B))
```

Comment:
```suggestion
	if(!isnull(B) && istype(B))
```
`istype()` is useful because it ensures that whatever value you have assigned to the value is indeed the type that you defined the variable as being. This will help you avoid runtime errors.